pilot/cmd/pilot-agent/main.go

```diff
@@ -237,6 +237,7 @@ var (
 				SecretTTL:                      secretTTLEnv,
 				SecretRotationGracePeriodRatio: secretRotationGracePeriodRatioEnv,
 			}
+			log.Infof("XDS auth provider is: %v", xdsAuthProvider.Get())
```

Comment:
Please remove, not relevant in most cases.

Reply:
Removed.